### PR TITLE
Add 'add_alias' function for using/documenting aliases

### PR DESCRIPTION
**Pull request by tduffield**
Signed-off-by: Tom Duffield <tom@chef.io>

### CODE REVIEW

File: bin/studio-common [hunk 1]
```diff
@@ -38,6 +39,17 @@ function document() {
   [ -z "$CI" ] && echo -e "$content" > "/tmp/docs/${name}"
 }
 
+# This is a helper function that allows you to quickly and easily add an alias for
+# a helper function. It will also ensure that a list of aliases are printed when
+# you run `describe`.
+function add_alias() {
+  fun="$1"
+  a="$2"
```
Comment (afiune) [inline bin/studio-common]: maybe some protection? (what if they don't provide any argument or just one?)

File: bin/studio-common [hunk 2]
```diff
@@ -38,6 +39,17 @@ function document() {
...
+
+  eval "alias $a=$fun"
+  [ -z "$CI" ] && echo -e "$a" > "/tmp/aliases/$fun"
```
Comment (afiune) [inline bin/studio-common]: Why not have aliases in CI systems? 🤔

Comment (tduffield) [reply bin/studio-common]: This is not _documenting_ the aliases. The alias is still being created on 49.

File: bin/studio-common (hunk 1 above)
Comment (tduffield) [reply bin/studio-common]: Sure. What type of behavior would you expect?

File: README.md [hunk 3]
```diff
@@ -140,6 +140,36 @@ The `document` function gives developers a quick and easy way to document the va
 
 The documentation provided via this function is collected and exposed via the `describe` command detailed below.
 
+#### `add_alias`
+
+The `add_alias` function performs double duty in that is will a) add the alias you specify, and b) automatically document that the alias exists.
```
Comment (afiune) [inline README.md]: [typo] The `add_alias` function performs double duty, it will a) ..., and b) ...

File: bin/studio-common (hunk 1 above)
Comment (afiune) [reply bin/studio-common]: Not needed honestly but I always ask myself questions like;
* What if they don't specify one argument and I eval `alias foobear=` or none `alias =` (I think one of them will blow up)
* What if I set the same alias to two functions? I would like to get a warning or block the action.

**Pull request merged**